stats/data_resolver.py

```diff
@@ -116,8 +116,8 @@ def get_conflicting_regions_by_count_of_involved_refactoring():
         project_rrs = rr_grouped_by_project.get_group(project_id)
         crh_rr_combined = pd.merge(project_crh.reset_index(), project_rrs.reset_index(), on='commit_hash', how='inner')
         crh_with_involved_refs = crh_rr_combined[crh_rr_combined.apply(record_involved, axis=1)]
-        crs_with_involved_refs = crs_with_involved_refs.append(crh_with_involved_refs.groupby('conflicting_region_id').size().to_frame())
-
+        #crs_with_involved_refs = crs_with_involved_refs.append(crh_with_involved_refs.groupby('conflicting_region_id').size().to_frame())
```

Comment:
Can you remove the commented out lines since you replaced them (applies to all changes)

Reply:
I just resolved your concerns with the new commit.